api/victoriametrics/v1beta1/additional.go

```diff
@@ -364,3 +364,11 @@ func (ds *DiscoverySelector) AsListOptions() (*client.ListOptions, error) {
 		LabelSelector: s,
 	}, nil
 }
+
+// ConfigMapKeyReference refers to a key in a ConfigMap.
```

Comment:
It's better to use default kubernetes type `ConfigMapKeySelector`

Reply:
@f41gh7 in the first version (where the files were formed by separate entries in Volumes[].Items and volumeMounts[]) i just used it, but after changes i became confused by the extra Optional field in this type, which is can't be used in any way